compiler/src/iree/compiler/Codegen/LLVMCPU/LLVMCPUVectorTransposeLowering.cpp

```diff
@@ -61,18 +61,25 @@ void LLVMCPUVectorTransposeLoweringPass::runOnOperation() {
     vectorTransformOptions.setVectorTransposeLowering(
         vector::VectorTransposeLowering::Shuffle16x16);
   }
+
+  constexpr unsigned specializedBenefit = 10;
+  constexpr unsigned narrowTypeEmulationBenefit = 20;
```

Comment:
minor nit: they should be

```suggestion
  constexpr unsigned kSpecializedBenefit = 10;
  constexpr unsigned kNarrowTypeEmulationBenefit = 20;
```

https://google.github.io/styleguide/cppguide.html#Constant_Names